src/classes/tenebrisScene.ts

```diff
@@ -0,0 +1,13 @@
+import { Scene } from "phaser";
+
+export class TenebrisScene extends Scene {
```

Comment:
All the scenes should be henceforth extended from `TenebrisScene` instead of `Scene`.

Reply:
Are we keeping the camelCase file naming convention? In that case, the Scenes (Boot.ts and TextScene.ts) have to be renamed. I am working on a refactoring PR rn, might as well change these.

i suggest we either keep the format like 'TenebrisScene.ts' or 'tenebris-scene.ts'.

Reply:
let it be `TenebrisScene.ts` this format